recipes/dagmc/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "dagmc" %}
+{% set version = "3.2" %}
+{% set sha256 = "becd61148aa4f9a3a312eb69e33d01779797007cc13715d60bb650ad5e71b183" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/svalinn/DAGMC/archive/v{{ version }}-rc.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler("fortran") }}
+  host:
+    - hdf5
+    - moab 5.*
+    - libblas
+    - liblapack
+  run:
+    - hdf5
+    - moab 5.*
+    - zlib
+
+test:
+   commands:
+     - make_watertight --help
+
+about:
+  home: https://svalinn.github.io/DAGMC/
+  license: BSD 2-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Direct Accelerated Geometry Toolkit'
+
+extra:
+  recipe-maintainers:
+    - gonuke
+    - pshriwise
+    - ljacobson64
+    - makeclean
+    - kkiesling
+    - bam241
+    - ngranda
```

Comment:
@gonuke @pshriwise @ljacobson64 @makeclean @kkiesling @bam241 are you OK being maintainers?

Reply:
Everyone other than @makeclean has indicated their willingness in comments above.

Reply:
Everyone other than @makeclean has indicated their willingness in comments above.